inc/assets.php

```diff
@@ -0,0 +1,173 @@
+<?php
+/**
+ * Contains functions for working with assets (primarily JavaScript).
+ *
+ * @package Archiveless
+ */
+
+namespace Archiveless;
+
+use Archiveless;
+
+define( 'ARCHIVELESS_ASSET_MAP', read_asset_map( dirname( __DIR__ ) . '/build/assetMap.json' ) );
+define( 'ARCHIVELESS_ASSET_MODE', ARCHIVELESS_ASSET_MAP['mode'] ?? 'production' );
+
+// Register action and filter hooks.
+add_action(
+	'enqueue_block_editor_assets',
+	__NAMESPACE__ . '\action_enqueue_block_editor_assets'
+);
+
+/*
+ * Unhook CSS and JS script concatenation on VIP Go.
+ *
+ * Script concatenation interferes with a lot of Gutenberg scripts—both ones
+ * that are written deliberately in this plugin, and ones that are written by
+ * WordPress itself in response to actions invoked via PHP. For safety's sake,
+ * we are unhooking both CSS and JS concatenation here. Script optimizations
+ * should be made at the level of the script build process in this plugin and
+ * in the theme, rather than relying on VIP's catch-all concatenation script.
+ */
+remove_action( 'init', 'css_concat_init' );
+remove_action( 'init', 'js_concat_init' );
```

Comment:
Recommend removing this from this plugin. Let sites declare this or not - this would be an unexpected side effect of including this plugin on a site.